katacomb/katacomb/conf/parameters/MKAT_narrow_L.yaml

```diff
@@ -8,14 +8,14 @@
 # Further details of the parameter selection can be found in JIRA ticket
 # SPR1-1332: https://skaafrica.atlassian.net/browse/SPR1-1332
```

Comment:
These are the other two tickets that cover this work:
https://skaafrica.atlassian.net/browse/SPR1-2202
https://skaafrica.atlassian.net/browse/SPR1-2774

Include the same on the offline config file also.

Reply:
I've added the tickets to the narrow band config file headers.